feat: add guide lines to README.md (#1611)
FromDoppler/doppler-webapp: README.md

To make use of this feature by console it can be enabled like this:
`localStorage.setItem('dopplerExperimental', JSON.stringify({PermissionHistory: true}));`

> ## Guide Lines

Reviewer: I think it would be better for this section to be at the beginning of the readme. So people read this first, and then the details.

### Test

- It's recommended to test from the user's point of view, hiding implementation details. [More details](https://testing-library.com/docs/).

- Recommended documentation for [test implementation](https://github.com/FromDoppler/doppler-webapp/pull/1504).

- Pre-commit hook is used in local development (husky+link-staged). [More details](https://github.com/FromDoppler/doppler-webapp/pull/1523).

- There's no QA team, so each development must be tested. Also, if a new development is not tested, coverage decreases. If the coverage is below the accepted limit, the merge will not be possible. [More details](https://github.com/FromDoppler/doppler-webapp/pull/1593).

### Code organization

> - **Components:** This is located in the *src/components* path. Each should be located in separate folders, each one must contain: *component implementation, tests, styles and images*. Also, the component is organized internally in the following order: hooks are located at the top, constants and states together in one place, next are the functions. Outside the component the variables or functions that do not depend on some props. [See slack conversation thread for more details](https://makingsense.slack.com/archives/CGJ72QFQX/p1625754680182000).

Reviewer: I think it would be more clear this way:
```suggestion
- **Components:** This is located in the *src/components* path. Each should be located in separate folders, each one must contain: *component implementation, tests, styles and images*. Also, the component is organized internally in the following order:
- hooks are located at the top
- constants and states together
- functions.
**Outside the component** the variables or functions that do not depend on some props. [See slack conversation thread for more details](https://makingsense.slack.com/archives/CGJ72QFQX/p1625754680182000).
```

Reviewer: ```suggestion
- **Components:** This is located in the *src/components* path. Each should be located in separate folders, each one must contain: *component implementation, tests, styles and images*. Also, the component is organized internally in the following order:
  - hooks are located at the top
  - constants and states together
  - functions.
**Outside the component** the variables or functions that do not depend on some props. [See slack conversation thread for more details](https://makingsense.slack.com/archives/CGJ72QFQX/p1625754680182000).
```
I've forgot to add an extra level for bullets 😅

Reviewer 2: Mmm, ahora mismo se está respetando todo esto?

Author: Bueno,  ahora mismo no veo múltiple niveles de identación.

Reviewer: Estamos intentando ir hacia ahí, en cada PR es algo que vamos revisando, si hay que acomodar varios componentes que no respetan ese formato. A eso hacías referencia @andresmoschini ?

- **Hooks:** This located in *src/hooks* path. Each should be located in separate folders, each one must contain: *hook implementation and tests*.

> - **Services:** This is located in the *src/services* path. Each should be located in separate folders, each one must contain: *service implementation, tests and service double*. 

Reviewer: I think this is more easy to read:
```suggestion
- **Services:** This is located in the *src/services* path. Each should be located in a separate folder and must contain: 
  - service implementation
  - tests
  - service double.
```

- **Translations:** This is located in *src/i18n* path. There are two files: *en.js* and *es.js*. If the translation is common, it's added within the *common* property, otherwise, it's added in the property that contains the translations of the corresponding module.

- **Gif/Images:** The reused gif/images are located in the *src/img* path. The particular images are located in the folder of the component that uses them. Example: a gif for the contact policy promotional page.

### Others

> - Use *let* and *const* insted of *var*

Reviewer 2: ```suggestion
- Use *let* and *const* instead of *var*
```

> - Use *let* and *const* insted of *var*

Reviewer 2: @jcamposmk crees que es posible poner algo sobre promover la inmuitabilidad? osea, que intentemos usar `const` en lugar de `let`?

Author: Suena bien :+1:  
Si tenés el enlace de algo que hable al respecto que pueda colocar para ampliar información sería de gran ayuda!
